### ISSUE TITLE
Format of signature

### ISSUE THREAD

**Issue by s1na** [austintgriffith/meta-transaction-format-share#6]
One of the differences I have seen among projects is how they transmit the signature of the meta tx, with some sending `v, r, s` fields separately, and some using an encoded version.

What are the arguments for each? For an encoded signature, which EIP does it adhere to ([EIP-191](https://github.com/ethereum/EIPs/issues/191) or [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md))?

**Comment by Kyrrui** [comment]
Yeah, we were passing the V, R, and S separately for the assumed increased gas costs. (We hadn't tested these assumptions to see the true cost of doing this operation on chain). Have not run into the "Stack Too Deep" error, but I would agree that if the cost is negligible then the standard should follow that we do the operation on chain for cleaner smart contract code readability and to Austin's point, to allow for more arguments.

**Comment by radek1st** [comment]
Same here, I just assumed it will cost more gas, but I haven't tested the exact amount. I've been hitting the "Stack Too Deep" sometimes, but normally walk around it by shuffling local variables.

**Comment by pet3r-pan** [comment]
Previous chat logs from the telegram group (archival purposes) https://hackmd.io/NPzA_EEFTuWR_DnnEEHOyw

**Comment by rmeissner** [comment]
If you have 1 signature it should be more gas efficient to pass the parts in separately (since `bytes` would be a dynamic type that is automatically copied to memory and therfore has some overhead). You can actually optimize this by hacking around a little in solitidy (external functions don't automatically copy their types into memory if not used).

The thing is that I would suggest to support multiple signatures and also standards like [EIP-1271](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1271.md). For this it is way more dynamic to use a `bytes`
